config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml

```diff
@@ -0,0 +1,66 @@
+periodics:
+- name: ci-kubernetes-e2e-kubeadm-kind-master
+  interval: 1h
+  labels:
+    preset-service-account: "true"
+    preset-bazel-scratch-dir: "true"
+    preset-bazel-remote-cache-enabled: "true"
+    preset-dind-enabled: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190205-d83780367-master
+      env:
+      # skip serial tests and run with --ginkgo-parallel
+      - name: "PARALLEL"
+        value: "true"
+      args:
+      - "--job=$(JOB_NAME)"
+      - "--root=/go/src"
+      - "--repo=k8s.io/kubernetes=master"
+      - "--repo=k8s.io/kubeadm=master"
+      - "--repo=sigs.k8s.io/kind=master"
+      - "--service-account=/etc/service-account/service-account.json"
+      - "--upload=gs://kubernetes-jenkins/logs"
+      - "--scenario=kubernetes_e2e"
+      - "--timeout=30m"
+      - "--"
+      - "--cluster="
+      - "--build=bazel"
+      - "--provider=skeleton"
+      - "--deployment=kind"
+      - "--kind-binary-version=latest"
```

Comment:
using this until we make a new release that includes some of the recent CLI features we've added e.g. `kind get nodes`

Reply:
soon! 😅 
maybe add a TODO?

Reply:
i will not forget to update this for all the jobs that will be added before that,.